accounting/src/main/java/io/skai/accounting/feign/WarehouseClient.java

```diff
@@ -9,6 +9,6 @@
         url = "http://localhost:8082/",
         configuration = FeignConfig.class)
 public interface WarehouseClient {
-    @RequestLine("POST /notifications/new-order")
-    String writeComponents(Map<Long, Integer> components);
+    @RequestLine("PUT /component")
```

Comment:
Request line is uncommon approach, please try   @RequestMapping(method = RequestMethod.GET, value = "/some-value")